qt/applications/workbench/workbench/projectrecovery/recoverygui/projectrecoverymodel.py

```diff
@@ -103,6 +103,7 @@ def recover_selected_checkpoint(self, selected):
             if isinstance(e, KeyboardInterrupt):
```

Comment:
I know it's not modified in this PR but this code is pointless, `KeyboardInterrupt` subclasses `BaseException` not `Exception` so `isinstance(e, KeyboardInterrupt)` is always `False`.

(see https://docs.python.org/3/library/exceptions.html)

Reply:
In Python 2.7 which we are currently running on. It is derived from StandardError which is in turn derived from Exception.

(See https://docs.python.org/2/library/exceptions.html)

Reply:
You are not wrong.